src/net.h

```diff
@@ -1396,6 +1396,9 @@ class CConnman
     // P2P timeout in seconds
     std::chrono::seconds m_peer_connect_timeout;
 
+    // Reachable networks that have addresses in the addrman
+    std::unordered_set<Network> networks_with_addrs;
```

Comment:
Should be [`m_networks_with_addrs`](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c)